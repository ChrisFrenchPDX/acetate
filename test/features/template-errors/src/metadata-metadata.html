---
description: Page with multiple metadata blocks
---

---
description: Page with multiple metadata blocks
---
{{ description }}