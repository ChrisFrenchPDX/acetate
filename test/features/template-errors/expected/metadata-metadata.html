---
description: Page with multiple metadata blocks
---
Page with multiple metadata blocks